swan-lake/learn-the-platform/run-in-the-cloud/code-to-cloud/code-to-cloud-deployment.md

```diff
@@ -88,6 +89,10 @@ Contains metadata about the Ballerina package. This file is used to fetch defaul
 cloud = "k8s" 
 ```
 
+***Config.toml***
+
+Contains the values which needs to be passed into configurable variables. You can read the [Configurable learn page](https://ballerina.io/learn/configure-ballerina-programs/configure-a-sample-ballerina-service/) to learn more about this. When you are using the Cloud features in ballerina, Config.toml is not being packed with container image as it can contain sensitive information. You need to explicitly provide this file in the runtime. 
```

Comment:
```suggestion
Contains the values, which need to be passed into the configurable variables. To learn more about this, see [Configure a sample Ballerina service](https://ballerina.io/learn/configure-ballerina-programs/configure-a-sample-ballerina-service/). When you are using the cloud-related features in Ballerina, the `Config.toml` file is not being packed with the container image as it can contain sensitive information. You need to explicitly provide this file in the runtime. 
```